core/client/fs/src/main/java/alluxio/client/block/stream/GrpcBlockingStream.java

```diff
@@ -92,8 +93,9 @@ public GrpcBlockingStream(Function<StreamObserver<ResT>, StreamObserver<ReqT>> r
    */
   public void send(ReqT request, long timeoutMs) throws IOException {
     if (mClosed || mCanceled || mClosedFromRemote) {
-      throw new CancelledException(formatErrorMessage(
-          "Failed to send request %s: stream is already closed or canceled.", request));
+      throw new CancelledException(
+          formatErrorMessage("Failed to send request %s: stream is already closed or canceled.",
```

Comment:
I didn't know what the correct spelling was, but this site says both are fine: https://www.grammarly.com/blog/canceled-vs-cancelled/

I changed it to `cancelled` because that is what the exception name is also using.